.github/workflows/e2e.yml

```diff
@@ -6,10 +6,10 @@ on:
       - dev
     tags:
       - v*
-  pull_request:
-    branches:
-      - dev
-      - master
+  # pull_request:
+  #   branches:
+  #     - dev
+  #     - master
```

Comment:
An issue with the test renderer and a missing global lib (TextEncoder) makes the tests that depend on the wallet-lib unable to run.

Reply:
After some investigation I was able to add the required global methods using a custom jest environment for tests.
It should be safe to run the tests now!